source/content/nutzen/api-nutzen.rst

```diff
@@ -1,154 +1,473 @@
 .. container:: custom-breadcrumbs
 
-   - :fa:`home` :doc:`Handbuch <../../index>` :fa:`chevron-right`
-   - :doc:`Nutzen <nutzen>` :fa:`chevron-right`
-   - opendata.swiss API nutzen
+    - :fa:`home` :doc:`Handbuch <../../index>` :fa:`chevron-right`
+    - :doc:`Nutzen <nutzen>` :fa:`chevron-right`
+    - opendata.swiss API nutzen
 
 *************************
 opendata.swiss API nutzen
 *************************
 
-Wie programmiere ich den Zugriff auf die Daten von opendata.swiss?
-==================================================================
+Wie nutze ich den API-Zugriff von opendata.swiss?
+=================================================
 
 .. container:: Intro
 
-   opendata.swiss basiert auf dem Open-Source-Projekt CKAN. CKAN stellt
-   eine umfangreiche API für die Metadaten des offenen Datenkatalogs zur Verfügung,
-   die Applikationsentwickler nutzen können. In diesem Kapitel erklären wir Ihnen,
-   wie Sie mittels API auf die Daten von opendata.swiss zugreifen können. Werfen
-   Sie auch einen Blick in
-   `aktuelle Beispiele, wie OGD in Anwendungen genutzt wird <https://opendata.swiss/showcase/>`__.
+    Das Portal `opendata.swiss <https://opendata.swiss/>`__ basiert auf dem Open-Source-Projekt CKAN. 
+    CKAN stellt eine umfangreiche API für die Metadaten des offenen Datenkatalogs zur Verfügung, die Applikationsentwickler_innen nutzen können. 
+    In diesem Kapitel erklären wir Ihnen, wie Sie mittels API auf die Daten von opendata.swiss zugreifen können. Werfen Sie auch einen Blick 
+    in aktuelle `Beispiele, wie OGD genutzt <https://opendata.swiss/showcase/>`__ wird.
 
-Nutzung der API – so funktioniert’s
+.. admonition:: Gut zu wissen
+
+    Hier geben wir Ihnen eine Einführung zur Nutzung der API von `opendata.swiss <https://opendata.swiss/>`__. 
+    Eine ausführliche Dokumentation zur CKAN API finden Sie in der `CKAN API Dokumentation <https://docs.ckan.org/en/latest/api/>`__.
+
+
+.. _api-nutzen-einführung:
+
+Einführung
+^^^^^^^^^^
+
+Eine `API (Application Programming Interface <https://de.wikipedia.org/wiki/Programmierschnittstelle>`__) ermöglicht es, dass Software direkt mit einem externen System (in diesem Fall den Metadaten auf der Plattform opendata.swiss) interagiert. Während die Webseite der Plattform opendata.swiss und die `Suchfunktion <https://opendata.swiss/de/dataset>`__ auf die Nutzung durch Menschen ausgerichtet ist, bieten wir über die API eine Zugriffsmöglichkeit, die für die Automatisierung optimiert ist.
```

Comment:
Please split line 32 to the several lines. 
From other content pages, it looks like Sabine was trying to keep text lines at around 76 chars
(but not more than 120 chars per line).